File: eFrizer/eFrizer.Model/TextReview/TextReviewInsertRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eFrizer.Model
{
    public class TextReviewInsertRequest
    {
        public int HairSalonId { get; set; }
        public int ClientId { get; set; }
        public string Text { get; set; }
    }
}

File: eFrizer/flutter_efrizer/lib/models/text_review/text_review_insert_request.dart
class TextReviewInsertRequest{
  final int hairsalonId;
  final int clientId;
  final String text;

  TextReviewInsertRequest({
    required this.hairsalonId,
    required this.clientId,
    required this.text,
  });

  Map<String, dynamic> toJson() => {
        "hairSalonId": hairsalonId,
        "clientId": clientId,
        "text": text,
      };
}